src/reader/arrow_transform.ts

```diff
@@ -156,21 +157,33 @@ export class ArrowRecordBatchTableRowTransform extends Transform {
   }
 }
 
-function convertArrowValue(fieldValue: any): any {
-  if (typeof fieldValue === 'object') {
-    if (fieldValue instanceof Vector) {
-      const arr = fieldValue.toJSON();
-      return arr.map((v: any) => {
-        return {v: convertArrowValue(v)};
-      });
-    }
-    const tableRow: TableRow = {f: []};
+function convertArrowValue(fieldValue: any, type: DataType): any {
+  if (fieldValue === null) {
+    return null;
+  }
+  if (DataType.isList(type)) {
+    const arr = fieldValue.toJSON();
+    return arr.map((v: any) => {
+      return {v: convertArrowValue(v, type.children[0].type)};
```

Comment:
Why do we only care about `type.children[0]`?

Reply:
Arrays/lists on BigQuery have the same datatype for every element, so getting the first one is enough here.